docs/csharp/programming-guide/statements-expressions-operators/default-value-expressions.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "default value expressions (C# Programming Guide)"
+
+ms.date: "2017-08-03"
+ms.prod: .net
+
+
+ms.technology: 
+  - "devlang-csharp"
+
+ms.topic: "article"
+dev_langs: 
+  - "CSharp"
+helpviewer_keywords: 
+  - "generics [C#], default keyword"
+  - "default keyword [C#], generic programming"
+ms.assetid: b9daf449-4e64-496e-8592-6ed2c8875a98
+caps.latest.revision: 22
+author: "BillWagner"
+ms.author: "wiwagn"
+
+translation.priority.ht: 
+  - "cs-cz"
+  - "de-de"
+  - "es-es"
+  - "fr-fr"
+  - "it-it"
+  - "ja-jp"
+  - "ko-kr"
+  - "pl-pl"
+  - "pt-br"
+  - "ru-ru"
+  - "tr-tr"
+  - "zh-cn"
+  - "zh-tw"
+---
+# default value expressions (C# programming guide)
+
+A default value expression produces the default value for a type. Default value expressions are particularly useful in generic classes and methods. One issue that arises using generics is how to assign a default value to a parameterized type `T` when you do not know the following in advance:
+
+- Whether `T` will be a reference type or a value type.
```

Comment:
nit: the future tense in these two sentences can be replaced with present tense.